text1A.js

```diff
@@ -1,17 +1,81 @@
 module.exports = {
   text: {
```

Comment:
Like I said, this is moving to the firebase, we can talk about how to structure this in the meeting on Saturday if you want but having this in the codebase doesn’t make sense for now